apps/cowswap-frontend/src/common/updaters/orders/UnfillableOrdersUpdater.ts

```diff
@@ -222,13 +221,8 @@ async function _getOrderPrice(chainId: ChainId, order: Order, strategy: PriceStr
   }
 
   const legacyFeeQuoteParams = quoteParams as LegacyFeeQuoteParams
-  // Limit order may have arbitrary validTo, but API doesn't allow values greater than 1 hour
-  // To avoid ExcessiveValidTo error we use PRICE_QUOTE_VALID_TO_TIME
-  if (order.class === 'limit') {
-    legacyFeeQuoteParams.validFor = Math.round(PRICE_QUOTE_VALID_TO_TIME / 1000)
-  } else {
-    legacyFeeQuoteParams.validTo = timestamp(order.validTo)
```

Comment:
`validTo` has been removed completely for quote requests. Always use `validFor` for everything now.
Quote responses still have it, as we use that as source of truth for current time.